src/ParametricOptInterface.jl

```diff
@@ -16,22 +16,22 @@ const PARAMETER_INDEX_THRESHOLD = Int64(4_611_686_018_427_387_904) # div(typemax
 const SIMPLE_SCALAR_SETS{T} =
     Union{MOI.LessThan{T},MOI.GreaterThan{T},MOI.EqualTo{T}}
 
-"""
-    Parameter(val::Float64)
+# """
+#     Parameter(val::Float64)
 
-The `Parameter` structure stores the numerical value associated to a given
-parameter.
+# The `Parameter` structure stores the numerical value associated to a given
+# parameter.
 
-## Example
+# ## Example
 
-```julia-repl
-julia> ParametricOptInterface.Parameter(5)
-ParametricOptInterface.Parameter(5)
-```
-"""
-struct Parameter <: MOI.AbstractScalarSet
-    val::Float64
-end
+# ```julia-repl
+# julia> MathOptInterface.Parameter(5.0)
+# MathOptInterface.Parameter(5.0)
+# ```
+# """
+# struct Parameter <: MOI.AbstractScalarSet
```

Comment:
To make this backwards compatible, you could do `const Parameter = MOI.Parameter{Float64}`?

Reply:
I am thinking about that still.
Maybe it is better just to break it sooner rather than later.

Reply:
Yeah maybe